starknet/src/space/space.cairo

```diff
@@ -41,13 +84,35 @@ trait ISpace<TContractState> {
         metadata_uri: Array<felt252>,
         dao_uri: Array<felt252>,
     );
+    /// Creates a new proposal. Users must go through an authenticator before calling this function.
+    ///
+    /// # Arguments
+    ///
+    /// * `author` - The author of the proposal.
+    /// * `metadata_uri` - The metadata URI of the proposal. Could be a link to a discussion page, etc.
+    /// * `execution_strategy` - The execution strategy of the proposal. This is the contract that will be called
```

Comment:
The execution strategy for the proposal, consisting of a strategy address and an execution payload.

Reply:
90a90ccc7f75db0456bd19d7cace99868b8f3f57

Reply:
7240ee167dd3f87aa02debe96a521af6b0b6cfd1